doc/release-notes/rl-2103.adoc

```diff
@@ -32,6 +32,8 @@ programs.broot.verbs = [
 ];
 ----
 
+* The option <<opt-programs.mako>> has been renamed to <<opt-services.mako>> and now provides a systemd user service.
```

Comment:
I think it's failing because it can't find options to link to (since the first one was removed).

I'd use `programs.mako` with backticks instead of `<< >>`, similar to how it's used here for sway: https://github.com/nix-community/home-manager/blob/b01f691eaf2330fdad4065759114bf593287aaf4/doc/release-notes/rl-2009.adoc

Reply:
That makes sense. What I don't quite understand is why it also fails to find `<<opt-services.mako>>`. Then again, I searched through doc and `<<opt-services.[...]>>` was apparently never used before, it's always single quotes or backticks. I guess I should just change that to `services.mako` aswell?